test/passing/js_source.ml.ref

```diff
@@ -4061,12 +4306,20 @@ let lexpr_ops (ops : ('a, 'a) #ops Lazy.t) =
   let lambda = lambda_ops ops in
   let expr = expr_ops ops in
   object (self : ([> 'a lexpr], 'a lexpr) #ops)
-    method free = function #lambda as x -> lambda#free x | #expr as x -> expr#free x
+    method free =
+      function
+      | #lambda as x -> lambda#free x
+      | #expr as x -> expr#free x
 
     method subst ~sub =
-      function #lambda as x -> lambda#subst ~sub x | #expr as x -> expr#subst ~sub x
+      function
+      | #lambda as x -> lambda#subst ~sub x
+      | #expr as x -> expr#subst ~sub x
 
-    method eval = function #lambda as x -> lambda#eval x | #expr as x -> expr#eval x
+    method eval =
```

Comment:
`function` could stay on the line before, similar to let bindings.

Reply:
Probably should go in another PR

Reply:
related to #462